### ISSUE TITLE
Class name on root DOM element triggering "Some page content is not contained by landmarks"

### ISSUE THREAD

**Issue by carterwcowan** [dequelabs/axe-core#2432]
Hi team,

We're integrating the axe-core library into all of our department's UI builds (yay its about time), and there's one error (Best Practice) that keeps coming up. Our site implements themeing by adding a css class to the root `<html>` element to cascade the theme through the rest of the page. 

Example: 
```html
<html class="dark">
```

While this css class does not necessarily add any styles to the `<html>` element itself other than setting a few css custom properties (see below)
```css 
:root.dark {
 --background-color: var(--dark-background-color) 
}
``` 
it does change the overall theme of the application. After running our builds, we've noticed that adding a class name to the `<html>` element is causing aXe to mark the css as "content" and as it is on the `<html>` and not contained by a landmark, we're shown the error "All page content must be contained by landmarks" (https://dequeuniversity.com/rules/axe/3.5/region?application=axeAPI).

**Expectation**: Adding a class name to the <html> element should not trigger the "Some page content is not contained by landmarks" error, as CSS is not considered "content"

**Actual**: Adding a class name to the <html> element causes the "Some page content is not contained by landmarks" error.

**Motivation**: According to WebAIM, "A primary benefit of CSS is that authors can separate content from its presentation. Content means the text, headings, list, regions, images, and other elements defined in HTML markup, along with their inherent semantics or meanings. Presentation means the way in which content is displayed as defined by CSS. By ensuring content is in HTML and presentation in CSS, if someone disables or overrides CSS, the content and semantics should still be fully accessible." https://webaim.org/techniques/css/

Thanks!

axe-core version: 3.5.5

**Comment by straker** [comment]
Thanks for the issue. Do you have a live example I could look at? Just adding a CSS class wouldn't trigger a failure of the `region` rule. Is the `dark` class being used to hide a region (perhaps by accident)?

**Comment by carterwcowan** [comment]
I'll take another look at the `dark` class. At the moment I'm not using it to change the display properties of any of the elements but you never know what could sneak in there.

Here's a simple codepen that doesn't seem to be triggering any errors, so perhaps mea culpa https://codepen.io/carterwcowan/pen/zYqOjxq
